app/views/Settings.js

```diff
@@ -114,35 +160,42 @@ class SettingsScreen extends Component {
   }
 
   getSettingRow(text, actionListener, icon, color, subtitle) {
-    const renderIcon = () => {
-      return icon ? <SvgXml xml={icon} /> : null;
+    const renderIcon = styles => {
+      return icon ? <SvgXml xml={icon} style={styles} /> : null;
     };
     return (
       <>
         <TouchableOpacity
           onPress={actionListener.bind(this)}
           style={styles.sectionRowContainer}>
-          {subtitle ? (
-            <Text
-              style={[
-                styles.settingRowText,
-                {
-                  color: color || Colors.VIOLET_TEXT,
-                  fontFamily: fontFamily.primaryBold,
-                },
-              ]}>
-              {text}
-            </Text>
-          ) : (
-            <Text
-              style={[
-                styles.settingRowText,
-                { color: color || Colors.VIOLET_TEXT },
-              ]}>
-              {text}
-            </Text>
-          )}
-          {renderIcon()}
+          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+            {renderIcon({
+              maxWidth: 24,
+              maxHeight: 24,
+              marginRight: 12,
+              marginTop: 2.5,
+            })}
+            {subtitle ? (
+              <Text
+                style={[
+                  styles.settingRowText,
+                  {
+                    color: color || Colors.VIOLET_TEXT,
```

Comment:
There is a theme aware `<Typography use="body1">` for this. Wrap the entire screen in `<Theme use="default">` and it all contained `<Typography>` will give you the violet text that you need.

If you need bold, you can `use="headline2"` or similar.